frontend/src/metabase/query_builder/components/DatasetEditor/ResizableNotebook/ResizableNotebook.jsx

```diff
@@ -18,10 +18,14 @@ function ResizableNotebook({
   resizableBoxProps,
   ...notebookProps
 }) {
+  {
+    /*reminder for me to handle width better*/
+  }
   return (
     <ResizableBox
       className="border-top flex"
       axis="y"
+      width={Infinity}
```

Comment:
Need to sort this out, the test doesn't pass without this - ResizableBox complains that no width prop was passed.